torch/_inductor/scheduler.py

```diff
@@ -2695,6 +2730,23 @@ def log_fusion(ms_fused: float, ms1: float, ms2: float) -> None:
                         red_text(f"{ms_fused / (ms1 + ms2):.3f}"),
                     )
 
+        async_compile = torch._inductor.async_compile.AsyncCompile()
+
+        def compile_kernel(
+            nodes: Sequence[BaseSchedulerNode],
+        ) -> tuple[Optional[TritonFuture], ModuleType]:
+            src_code = self.generate_kernel_code_from_nodes(
+                nodes, benchmark_kernel=True
+            )
+            mod = PyCodeCache.load(src_code)
+            if not async_compile.use_process_pool():
+                fut = None
```

Comment:
Does this fall back to serial compilation?

Reply:
yes